docs/commit-log.rst

```diff
@@ -30,34 +21,25 @@ following are rules to follow when committing a change to the git repo:
    wrap long lines.  If you don't, the 'git log' output ends up looking
    stupid on 80 column terminals.
 
-3) For RHEL bugs, all commits need to reference a bug number.  You may
-   follow one of two formats for specifying the bug number in a RHEL commit.
-
-   a)  Put the bug number on the summary line in (#BUGNUMBER) format.  Bugs
-       listed this way are treated as 'Resolves' patches in the RHEL
-       universe.
-
-   b)  If you have a patch that is Related to or Conflicts with another bug,
-       you may add those lines to the end of the long commit message in this
-       format::
+3) For RHEL bugs, all commits need to reference a bug issue name.
```

Comment:
You can mention that both RHEL and CentOS Stream contributions need a bug ID.
Also, point the users to https://issues.redhat.com/projects/RHEL/issues as a place where to report such bugs.

Reply:
Fixed.